app/controllers/submission.js

```diff
@@ -388,7 +396,16 @@ function curateGenesAndAnnotations(req, res) {
             });
 
             // Now that the request is valid, start the update.
-            return performSubmissionOrCuration(req, validationResult, curSubmission);
+            return performSubmissionOrCuration(req, validationResult, curSubmission).then(() => {
+
+                if (req.body.annotations.every(newAnnotation => {
+                    return newAnnotation.status != 'pending';
+                })) {
+                    //When we get emails, replace hard coded email with submitter email
+                    sendEmail.createMessage("ajr1644@rit.edu", "Cur");
```

Comment:
Write the code to get the submission user's email here, this is different than when the user is submitting because you have to get the user object from the existing submission's relation.

Reply:
Addressed in upcoming push